### PR TITLE
nix-prefetch-hg: Various bash style improvements

### PR DESCRIPTION
**Pull request by benley**

### CODE REVIEW

File: pkgs/build-support/fetchhg/nix-prefetch-hg [hunk 1]
```diff
@@ -4,79 +4,79 @@ url=$1
...
-hashType=$NIX_HASH_ALGO
-if test -z "$hashType"; then
-    hashType=sha256
-fi
-if test -z "$hashFormat"; then
-    hashFormat=--base32
-fi
+hashType="${NIX_HASH_ALGO:-sha256}"
+hashFormat=${hashFormat:-"--base32"}
+rev="${rev:-tip}"
```
Comment (vcunat) [inline pkgs/build-support/fetchhg/nix-prefetch-hg]: Any particular reason why not change the `rev` assignment above instead? (I'm sorry the PR was sitting here for so long.)

**Comment by vcunat** [comment]
It looks good to me. I assume you have tested it, at least a bit?

**Comment by garbas** [comment]
:+1: tested and merging

File: pkgs/build-support/fetchhg/nix-prefetch-hg (hunk 1 above)
Comment (benley) [inline pkgs/build-support/fetchhg/nix-prefetch-hg]: No reason, just visual style I think.  I totally forgot about this PR!

**Pull request closed without merging**